test/unittests/test_recordstore.py

```diff
@@ -457,6 +461,9 @@ def test_create_project(self):
     def test_project_info(self):
         self.assertEqual(self.store.project_info("TestProject")["name"], "TestProject")
 
+    def test_clear(self):
+        pass  # raises UserWarning
```

Comment:
Based on the comment I first thought that the purpose of the `pass` statement was to raise a `UserWarning`, which confused me. I'd suggest changing the comment to "override base class test to avoid UserWarning" or something along those lines.